portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includePage.vm

```diff
@@ -29,10 +29,13 @@
     #set( $numberTools = $pageColumn0Tools.size() )
 #end
 <div class="portal-main-container container-fluid mt-2">
-    <main id="$pageWrapperClass" class="portal-main-content d-flex flex-column #if( $numberTools > 1 || $homePage)Mrphs-multipleTools #end" role="main">
+    <main id="$pageWrapperClass" class="portal-main-content #if( $numberTools > 1 || $homePage)Mrphs-multipleTools #end" role="main">
         <h2 class="skip visually-hidden" tabindex="-1" id="tocontent">${rloader.sit_contentshead}</h2>
         #parse("/vm/morpheus/includeSiteHierarchy.vm")
         #parse("/vm/morpheus/snippets/siteStatus-snippet.vm")
+        #if ($pageTwoColumn)
+        <div class="d-flex">
+        #end
```

Comment:
```suggestion
          <div class="${pageTwoColumn ? 'd-flex' : ''}">
```

Reply:
Hello, @kunaljaykam . I'm open to your approach, but the velocity parser does not seem to support ternary operators. As is, the page does not load w/ a runtime error:

org.apache.velocity.exception.ParseErrorException: Encountered " ? \'d-flex\' : \'\'}\">\n            <div id=\"" at /vm/morpheus/includePage.vm[line 36, column 36]
Was expecting one of:
    "}" ...
    <DOT> ...

The closest approximation I could find for a ternary syntax in Velocity requires another #set statement like:

        #set( $flexClass = "#if(${pageTwoColumn})d-flex#end" )
        <div class="${flexClass}">

While the above actually works, I think it sacrifices code legibility, especially since the prior conditional I proposed mirrors the same logic and style elsewhere in the velocity template. Furthermore, this approach adds an extraneous div to the DOM for single column layouts.

That said, let me know how you'd like to proceed.

Reply:
Well, it was worth a try. Thank you so much Sean for considering all of my suggestions. :+1: